packages/sui-mock/README.md

```diff
@@ -0,0 +1,135 @@
+# sui-mock
+
```

Comment:
What's the main purpose of the package?
I would say: 
It is mainly a wrapper around [Mock Service Worker (MSW)](https://mswjs.io/), it allows mocking by intercepting requests on the network level. Seamlessly reuse the same mock definition for testing, development, and debugging.